src/Materials/fresnelParameters.ts

```diff
@@ -28,22 +89,32 @@ export class FresnelParameters {
     /**
      * Define the color used on edges (grazing angle)
      */
-    public leftColor = Color3.White();
+    public leftColor: Color3;
 
     /**
      * Define the color used on center
      */
-    public rightColor = Color3.Black();
+    public rightColor: Color3;
 
     /**
      * Define bias applied to computed fresnel term
      */
-    public bias = 0;
+    public bias: number;
 
     /**
      * Defined the power exponent applied to fresnel term
      */
-    public power = 1;
+    public power: number;
+
+    public constructor(options: IFresnelParametersCreationOptions = {}) {

```

Comment:
You need to add a comment on the constructor as we do in the other classes for the build to be ok :-)